astronomer/providers/amazon/aws/example_dags/example_redshift_sql.py

```diff
@@ -81,12 +107,25 @@
         redshift_conn_id=REDSHIFT_CONN_ID,
     )
 
+    delete_redshift_cluster = BashOperator(
+        task_id="delete_redshift_cluster",
+        bash_command=f"aws redshift delete-cluster "
+        f"--cluster-identifier {REDSHIFT_CLUSTER_IDENTIFIER} --skip-final-cluster-snapshot && sleep 2m",
```

Comment:
Same here explicit sleep for 2 minutes. We could use boto3 to poll for cluster status.

Reply:
Address here a42720f3827ed0b4ae890883f4470f0f88125b61